apps/communications/contacts/test/unit/views/multiple_select_test.js

```diff
@@ -0,0 +1,95 @@
+/* global Contacts, MockContactsList, MockLazyLoader, MockMultipleSelectHTML */
+'use strict';
+
+requireApp('communications/contacts/js/views/multiple_select.js');
+requireApp('communications/contacts/test/unit/mock_contacts_list.js');
+require('/shared/test/unit/mocks/mock_lazy_loader.js');
+requireApp('communications/contacts/test/unit/mock_multiple_select.js.html');
+
+window.LazyLoader = window.LazyLoader || {};
```

Comment:
please mock these objects properly. if you are going to Mock, mock them actually, do not use that pattern